lib/screens/qr_code_screen.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:superformula_qr_code/models/seed_model.dart';
+import 'package:superformula_qr_code/services/api_service.dart';
+import 'package:superformula_qr_code/shared/app_routes.dart';
+import 'package:http/http.dart' as http;
+
+class QrCodeScreen extends StatefulWidget {
```

Comment:
**bug:** when I first loaded this view, I was presented with a timer and a spinner. I do not have the server running locally. I would have expected an error due to that.
![Simulator Screen Shot - iPhone 13 - 2021-11-23 at 14 26 51](https://user-images.githubusercontent.com/53348890/143090864-9badb8db-7b20-4ca4-8716-b5b1a1252cf9.png)